locales/es.js

```diff
@@ -0,0 +1,155 @@
+export const es = {
+  welcome: {
+    title: 'Abra su nueva cuenta bancaria',
+    description_p_1: 'Para abrir una cuenta bancaria, necesitaremos verificar su identidad.',
+    description_p_2: 'Solo tomará un par de minutos.',
```

Comment:
Some older users might think this is a typo: http://www.tallerdeletras.com/la-rae-sola-contra-el-acento-del-adverbio-solo/ (I thought it should have been Sólo).